app/models/location.rb

```diff
@@ -1,5 +1,5 @@
 class Location < ApplicationRecord
-  has_operation :geocoding
+  has_operation :geocoding, class_name: "Geocoding::V1"
```

Comment:
I like that this has the flexibility to name the operations several different ways if need be.